Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -421,6 +421,147 @@ public IEnumerator NearInteractionTouchableUnityUiToggle()
             UnityEngine.Object.Destroy(canvas.gameObject);
         }
 
+        /// <summary>
+        /// Test minimum touch distance.
+        /// </summary>
+        /// <remarks>
+        /// Near interaction uses two separate distance calculations:
+        /// 1. The PokePointer uses the DistanceToTouchable to determine the closest touchable. This distance value is also used compared to PokePointer.TouchableDistance to determine the set of eligible touchables in the first place.
+        ///     There is no explicit spec which space this distance is computed in, although it should be in world space. Due to the way touchables convert pointer position into local space first and then do a scale-only inverse transform there may be errors.
+        /// 1. After the PokePointer has selected the closest touchable, it then performs a raycast against either the collider or the UnityUI canvas, depending on type. The (world space) length of that ray is then used as the actual distance for triggering touch events.
+        ///     When the ray length is _less_ than the PokeThreshold the TouchDown event is raised and the touchable is "down".
+        ///     When the touchable is "down" and the ray length is _greater_ than PokeThreshold + DebounceThreshold (!) the TouchUp event is raised and the touchable is released.
```

Comment:
That's ok, it just wasn't clear that the debounce threshold is actually added onto the PokeThreshold rather than used as a separate distance from the collider surface.